From: Joachim Bauch <bauch@struktur.de>
Subject: Fix detection of OpenJPEG 2 in Debian.
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/1071999

Index: libheif/CMakeLists.txt
===================================================================
--- libheif.orig/CMakeLists.txt
+++ libheif/CMakeLists.txt
@@ -211,7 +211,7 @@ endif()
 plugin_option(OpenJPEG_ENCODER "OpenJPEG J2K encoder" OFF ON)
 plugin_option(OpenJPEG_DECODER "OpenJPEG J2K decoder" OFF ON)
 if (WITH_OpenJPEG_ENCODER OR WITH_OpenJPEG_DECODER)
-    find_package(OpenJPEG 2)
+    find_package(OpenJPEG)
 endif()

 # ffmpeg
